content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,49 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the time when you first added the Docsy submodule, 
+or last updated. If you have any modified content like a Docsy [customiztion](/docs/adding-content/lookandfeel/), those 
+changes will remain in your project. For details about what has changed, see the list of 
+[Docsy commits](https://github.com/google/docsy/commits/master).
+
+## Update a Docsy clone
+
+If you have a local clone of the Docsy theme, you `git pull` all the new changes. For any modifications that you
+made to the theme, you must manually resolve any merge conflicts. 
+
+For example, if `upstream`
+is your `remote` that is set to `https://github.com/google/docsy.git`, then from the root 
+directory where you cloned the Docsy theme, you run:
+
+```
+git pull upstream master
+```
+
+## Update a Docsy submodule
+
+If you are using the Docsy theme as a submodule in your project, you update the submodule:
+
+1. Navigate to the root of your local project, then run:
 
     ```
     $ git submodule update --remote
     ```
     
-1. Then add and commit your change:
+1. Add and then commit the change to your project:
 
     ```
     $ git add themes/
     $ git commit -m "Updating theme submodule"
     ```
 
-1. Finally, push the change back to the project repo.
+1. Push the commit to your repo. For example, if your `remote` is `origin`, then you run: 
```

Comment:
The "if your remote is origin" bit seems a bit odd, if you've made a local copy of your repo and you're doing this work in it, the relevant remote is "origin" by default, you'd have to have explicitly set it to something else.

How about:
Push the commit to your project repo (by default this is your `origin` remote if you made a local copy of the repo):

Reply:
Per our chat, let me know if `origin` is set by default since im not getting  that behavior. I also intentionally left this vague since someone could have set their remote name as `docsy` or any other unique name.

Reply:
Yes, it is - I just tried cloning the Kubeflow website (which I don't own) into a new directory on my Mac and origin is set to the Kubeflow website.

Reply:
Sorry! And thanks for trying this out for me! I tried again too and found that the correct command to list the remote and url is `git remote get-url origin` (where `git remote` only list the remote without the URL, but I incorrectly read that as "`origin` has no URL set"). 

```suggestion
1. Push the commit to your project repo (by default this is your `origin` remote):
```